pyradiosky/skymodel.py

```diff
@@ -625,6 +857,8 @@ def __init__(
 
             if name is not None:
                 self.name = np.atleast_1d(name)
+            if skycoord is not None:
+                self.skycoord = skycoord
```

Comment:
So this catches the case that coordinates are passed in and need to be converted to a skycoord. Since we make those, we make them not to be scalars, but I think you're right that we shouldn't double assign, so I'll remove the assignment on line 723.